plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -106,7 +104,7 @@ private static String convertOrderByExpressionToPql(OrderByExpression orderByExp
     {
         requireNonNull(orderByExpression, "orderByExpression is null");
         StringBuilder builder = new StringBuilder()
-                .append(quoteIdentifier(orderByExpression.getColumn()));
+                .append(orderByExpression.getExpression());
```

Comment:
Is this quoted somewhere?

Reply:
Yes, the OrderByExpression is created from the PinotColumnHandle.getExpression which is quoted.